### PR TITLE
Add FAQ about docker snap package

### LINKED ISSUE #78: Error executing sourced init when docker is installed using snap
This is the error:
```
./sourced init /home/antonio/bblfsh-org/
docker-compose working directory set to /home/antonio/.sourced/workdirs/home/antonio/bblfsh-org
Traceback (most recent call last):
  File "/snap/docker/384/bin/docker-compose", line 11, in <module>
    load_entry_point('docker-compose==1.22.0', 'console_scripts', 'docker-compose')()
  File "/snap/docker/384/lib/python2.7/site-packages/compose/cli/main.py", line 71, in main
    command()
  File "/snap/docker/384/lib/python2.7/site-packages/compose/cli/main.py", line 124, in perform_command
    project = project_from_options('.', options)
  File "/snap/docker/384/lib/python2.7/site-packages/compose/cli/command.py", line 26, in project_from_options
    environment = Environment.from_env_file(project_dir)
  File "/snap/docker/384/lib/python2.7/site-packages/compose/config/environment.py", line 61, in from_env_file
    instance = _initialize()
  File "/snap/docker/384/lib/python2.7/site-packages/compose/config/environment.py", line 57, in _initialize
    return cls(env_vars_from_file(env_file_path))
  File "/snap/docker/384/lib/python2.7/site-packages/compose/config/environment.py", line 35, in env_vars_from_file
    with contextlib.closing(codecs.open(filename, 'r', 'utf-8-sig')) as fileobj:
  File "/snap/docker/384/usr/lib/python2.7/codecs.py", line 896, in open
    file = __builtin__.open(filename, mode, buffering)
IOError: [Errno 13] Permission denied: u'./.env'
exit status 1
```

### PR DESCRIPTION
**Pull request by carlosms**
Close #78.

---

* [ ] I have updated the CHANGELOG file according to the conventions in [keepachangelog.com](https://keepachangelog.com)
* [x] This PR contains changes that do not require a mention in the CHANGELOG file

### CODE REVIEW

**Review by smacker — APPROVED** [review]

File: docs/faq.md [hunk 1]
```diff
@@ -137,3 +137,7 @@ the [Architecture documentation](./architecture.md#docker-networking)
 The user interface is based in the open-sourced [Apache Superset](http://superset.apache.org),
 so you can also refer to [Superset tutorials](http://superset.apache.org/tutorial.html)
 for advanced usage of the web interface.
+
+## I Get IOError: [Errno 13] Permission denied: u'./.env'
```
Comment (dpordomingo) [inline docs/faq.md]: This link may not work both in git and GotBook. Could you try in GitBook?
What if using something like the following instead?
```suggestion
## I Get IOError Errno 13 Permission denied
```

File: docs/faq.md [hunk 2]
```diff
@@ -137,3 +137,7 @@ the [Architecture documentation](./architecture.md#docker-networking)
...
+
+This may happen if you have installed Docker from a snap package. This installation mode is not supported, please install it following [the official documentation](https://docs.docker.com/install/) (See [#78](https://github.com/src-d/sourced-ce/issues/78)).
```
Comment (dpordomingo) [inline docs/faq.md]: What if linking our docs about how to install docker? We already provide some links to the different OS.

**Review by dpordomingo — COMMENTED** [review]
What if we also mention this limitation when explaining source{d} dependencies?
https://github.com/src-d/sourced-ce/blob/master/docs/quickstart/1-install-requirements.md#install-docker
I'd mention it when saying that docker toolbox is not allowed.

**Review by se7entyse7en — APPROVED** [review]

**Comment by carlosms** [comment]
@dpordomingo I have pushed changes for all you suggestions

**Review by dpordomingo — APPROVED** [review]
LGTM. Many thanks @carlosms

**Pull request merged**